localstack/services/dynamodb/provider.py

```diff
@@ -450,11 +456,15 @@ def delete_expired_items() -> int:
     return no_expired_items
 
 
-def _get_partition_key(table_description: DescribeTableOutput) -> str:
+def _get_hash_and_range_key(table_description: DescribeTableOutput) -> [str, str | None]:
```

Comment:
I don't think this should be an issue, since we retrieve the keys for each distinct table and ddblocal should make sure we have a valid key schema here.